config/config.go

```diff
@@ -361,6 +362,19 @@ type JobDebugSettings struct {
 	RPCLog bool `yaml:"rpc_log,optional,default=false"`
 }
 
+type HookList []HookSettings
+
+type HookSettings struct {
```

Comment:
Each hook type should be its own Go type. Look at `PruningEnum`:

https://github.com/zrepl/zrepl/blob/afed7627749bee3726a06f7855a11a82b6769b09/config/config.go#L271-L273

https://github.com/zrepl/zrepl/blob/afed7627749bee3726a06f7855a11a82b6769b09/config/config.go#L416-L424

 https://github.com/zrepl/zrepl/blob/afed7627749bee3726a06f7855a11a82b6769b09/config/config.go#L153-L155